ontology/entity_extraction.ipynb

```diff
@@ -0,0 +1,636 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.            unwanted_tokens = (
</pre><p>Overall looks great. Can we investigate more on unwanted tokens? Not sure how freely it can be done. Some gooe [website](https://hanlp.hankcs.com/docs/annotations/pos/ud.html)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nicklin37/kg-via-dl/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
[website](https://hanlp.hankcs.com/docs/annotations/pos/ud.html)